recipe/meta.yaml

```diff
@@ -8,14 +8,17 @@ source:
   sha256: 1524b840e42cf3b909e8f8df67c1724012c7dc7f9d076d4feef2d3eff031e8a0
   patches:
       - omit-local-site-packages.patch  # [osx]
+      - osx64-dist.patch                # [osx and x86_64]
+      - win-find_exe.patch              # [win]
+      - win-library_bin.patch           # [win]
 
 build:
-  number: 1
+  number: 2
   # Windows has issues updating python if conda is using files itself.
   # Copy rather than link.
   no_link:
-    - DLLs/_ctypes.py   # [win]
-    - bin/python
+    - DLLs/_ctypes.pyd  # [win]
+    - bin/python3.5     # [unix]
```

Comment:
Both of those lines were taken from the [python 3.5 recipe on anaconda-recipes](https://github.com/ContinuumIO/anaconda-recipes/blob/master/python-3.5/meta.yaml)

bin/python is a symbolic link to bin/python3.5 so it makes more sense to not link the true file as opposed to the symbolic link. 

DLLs/_ctypes.py does not exist and it should be _ctypes.pyd, just a typo in the original recipe that was never caught.  I believe it must be because of the comment above in the souce.